src/hammer-vlsi/defaults_types.yml

```diff
@@ -0,0 +1,982 @@
+# Core configuration options and defaults for hammer-vlsi.
+# The values specified in this file are the defaults.
+# e.g. foo: "bar" in this file means that the default setting for foo is "bar".
+
+vlsi.core:
+  # Path(s) to out-of-tree hammer technology libraries.
```

Comment:
In general, I don't think it's necessary to mirror all the comments, especially all the allowable values/TODOs in this types file.